Composer/packages/client/src/utils/botRuntimeUtils.ts

```diff
@@ -12,14 +12,17 @@ export const getBotStatusText = (currentBotStatus) => {
       return BotStatusesCopy.published;
 
     case BotStatus.reloading:
-      return BotStatusesCopy.loading;
+      return BotStatusesCopy.reloading;
 
     case BotStatus.connected: {
       return BotStatusesCopy.connected;
     }
     case BotStatus.publishing:
       return BotStatusesCopy.publishing;
 
+    case BotStatus.queued:
+      return BotStatusesCopy.queued;
+
     default:
     case BotStatus.unConnected:
       return BotStatusesCopy.unConnected;
```

Comment:
Wouldn't `BotStatusesCopy[currentBotStatus] ?? BotStatusesCopy.unConnected` handle all of these cases?

Reply:
I think so. I'll do the change.